### ISSUE TITLE
Add workarounds for the workshop value restrictions

### ISSUE THREAD

**Issue by 2020snc** [Zezombye/overpy#182]
looks like `OR` doesn't take `Throttle Of` as a valid value

overpy code
```python
rule "1":
    @Event eachPlayer
    @Condition eventPlayer.getThrottle() != vect(0,0,0) or eventPlayer.isHoldingButton(Button.ABILITY_1)
```

workshop code
```
rule ("1") {
    event {
        Ongoing - Each Player;
        All;
        All;
    }
    conditions {
        Or(Throttle Of(Event Player), Is Button Held(Event Player, Ability 1)) == True;
    }
}
```
Using `#!disableOptimizations` will temporarily solve my problem

**Comment by Zezombye** [comment]
This is fixed in the next release, however this does not fix the underlying issue (which is that `Throttle Of` and similar functions should be able to be put in a boolean value, eg `if (eventPlayer.getThrottle())` should work).

I am leaving this open while this underlying bug is not fixed.

**Comment by Zezombye** [comment]
Fixed in 6.6.6: all values can now be put in boolean fields. If the workshop doesn't accept the raw value, it will be wrapped with `First Of`.
